lib/features/email/data/datasource/calendar_event_datasource.dart

```diff
@@ -1,11 +1,20 @@
 
 import 'package:jmap_dart_client/jmap/account_id.dart';
 import 'package:jmap_dart_client/jmap/core/id.dart';
-import 'package:jmap_dart_client/jmap/mail/calendar/calendar_event.dart';
+import 'package:jmap_dart_client/jmap/mail/calendar/reply/calendar_event_accept_response.dart';
+import 'package:jmap_dart_client/jmap/mail/calendar/reply/calendar_event_maybe_response.dart';
+import 'package:jmap_dart_client/jmap/mail/calendar/reply/calendar_event_reject_response.dart';
 import 'package:tmail_ui_user/features/email/domain/model/event_action.dart';
+import 'package:tmail_ui_user/features/email/presentation/model/blob_calendar_event.dart';
 
 abstract class CalendarEventDataSource {
-  Future<Map<Id, List<CalendarEvent>>> parse(AccountId accountId, Set<Id> blobIds);
+  Future<List<BlobCalendarEvent>> parse(AccountId accountId, Set<Id> blobIds);
 
   Future<List<EventAction>> getListEventAction(String emailContents);
```

Comment:
I found we deleted it in `repository` but still not delete it in `datasource`

Reply:
Removed